docs/docs/60-new-docs/50-user-guide/20-how-to-guides/05-installing-kargo-cli.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: Installing Kargo CLI
+---
+
+# Installing Kargo CLI
+
+The Kargo CLI provides a command-line interface to manage Kargo efficiently, serving as
+a faster, automation-friendly alternative to the UI.
+
+Choose the installation method for your operating system:
+
+<Tabs groupId="os">
+<TabItem value="mac-linux-wsl" label="Mac, Linux, or WSL" default>
+
+1. Run the following command to download the CLI:
+    ```shell
+    arch=$(uname -m)
+    [ "$arch" = "x86_64" ] && arch=amd64
+    curl -L -o kargo https://github.com/akuity/kargo/releases/latest/download/kargo-"$(uname -s | tr '[:upper:]' '[:lower:]')-${arch}"
+    chmod +x kargo
+    ```
+
+    _Alternatively_, you can install the Kargo CLI through the <Hlt>CLI</Hlt> Tab in the Kargo UI:
```

Comment:
imho, we should reverse the order of these. i.e. Promote the UI as the best option. Something particularly nice about that is the versions downloaded from the UI will match the version of Kargo that's installed in the cluster, which is a plus.

I also think it's easier.

Reply:
On that note, I think we should _also_ update the quickstart to promote downloading from the UI. There, I think we can get away without even including the manual instructions because if the reader reaches that step of the quickstart, we know with certainty that the server side components of Kargo are installed and we even know the address of the download page.

Could you please bundle that change into this PR?